MekHQ/src/mekhq/campaign/io/CampaignXmlParser.java

```diff
@@ -1802,8 +1802,9 @@ private static boolean isLegacyMissingMASC(Part p) {
     }
 
     private static void updatePlanetaryEventsFromXML(Node wn) {
-        // TODO: make Planets a member of Campaign
-        Planets.reload(true);
+        // TODO: FIX THIS for new structure
+        /*
```

Comment:
Let's figure out what to do with this commented out code.

Reply:
I have implemented this in commit 7d72dde.